src/arch/armv8/vgic.c

```diff
@@ -779,35 +819,22 @@ struct vgic_reg_handler_info ipriorityr_info = {
     vgic_int_set_prio_hw,
 };
 
-struct vgic_reg_handler_info vgicd_misc_info = {
-    vgicd_emul_misc_access,
-    0x4,
-};
+struct vgic_reg_handler_info vgicd_misc_info = { vgicd_emul_misc_access, 0x4, 0, 0, 0, NULL, NULL,
+    NULL };
```

Comment:
We can add a comma after the last `NULL` to force the formatted to put one field in each line.

Reply:
The same for the rest vgic info structs..

Reply:
Fixed in 2d0aea7612bcb21d087afdf77c66b2e835a0061b